connector/connect/common/src/test/resources/query-tests/explain-results/function_decode.explain

```diff
@@ -1,2 +1,2 @@
-Project [decode(cast(g#0 as binary), UTF-8, false) AS decode(g, UTF-8)#0]
+Project [decode(cast(g#0 as binary), UTF-8, false, false) AS decode(g, UTF-8)#0]
```

Comment:
shall we exclude these two flags from the pretty string of the expression?

Reply:
OK